nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```diff
@@ -271,6 +331,23 @@ in {
         }
       ]) (builtins.attrNames cfg.extraFiles);
 
+    warnings = lib.optional cfg.secureBoot.enable
+    ''You enabled Lanzaboote's experimental SecureBoot implementation.
```

Comment:
Formatting.
```suggestion
    warnings = lib.optional cfg.secureBoot.enable ''
      You enabled Lanzaboote's experimental SecureBoot implementation.
```

Reply:
This is important otherwise there might be extra spaces in front of the warning which shouldn't be there